arelle/FunctionIxt.py

```diff
@@ -1108,7 +1149,6 @@ def numunitdecimalin(arg):
     # same as v2: 'zerodash': zerodash,
 })
     # transformation registry v-4 functions
-tr4Functions = tr3Functions.copy() # tr4 starts with tr3 and adds more functions
```

Comment:
Why this change? Does TR4 not inherit things from TR3 any more?

Reply:
Never did, that was a goof.